doc/releases/changelog-0.27.0.md

```diff
@@ -4,145 +4,235 @@
 
 <h3>New features since last release</h3>
 
-<h4>(TODO: title) JAX JIT</h4>
+<h4>An all-new data module 💾</h4>
 
-<h4>(TODO: title) Qutrits</h4>
+* The `qml.data` module is now available, allowing users to download, load, and create quantum datasets.
+  [(#3156)](https://github.com/PennyLaneAI/pennylane/pull/3156)
 
-* The `qml.GellMann` qutrit observable, the ternary generalization of the Pauli observables, is now available.
-  ([#3035](https://github.com/PennyLaneAI/pennylane/pull/3035))
+  Datasets are hosted on Xanadu Cloud and can be downloaded by using `qml.data.load()`:
 
-  When using `qml.GellMann`, the `index` keyword argument determines which of the 8 Gell-Mann matrices is used.
+  ```pycon
+  >>> H2_datasets = qml.data.load(
+  ...   data_name="qchem", molname="H2", basis="STO-3G", bondlength=1.1
+  ... )
+  >>> H2data = H2_datasets[0]
+  >>> H2data
+  <Dataset = description: qchem/H2/STO-3G/1.1, attributes: ['molecule', 'hamiltonian', ...]>
+  ```
 
-  ```python
-  dev = qml.device("default.qutrit", wires=2)
+  - Datasets available to be downloaded can be listed with `qml.data.list_datasets()`.
 
-  @qml.qnode(dev)
-  def circuit():
-      qml.TClock(wires=0)
-      qml.TShift(wires=1)
-      qml.TAdd(wires=[0, 1])
-      return qml.expval(qml.GellMann(wires=0, index=8) + qml.GellMann(wires=1, index=3))
-  ```
+  - To download or load only specific properties of a dataset, we can specify the desired properties in `qml.data.load` with the `attributes` keyword argument:
```

Comment:
to be honest, slight implementation secret but `attributes` only affects what's _downloaded_. The way `load` works is:
1. Determine all descriptions (combinations of parameters other than `attributes`) the user is requesting
2. Download one file per-attribute, per-description (the default "full" attribute gets a "_full.dat" file) to a folder following a naming format (eg. `datasets/qchem/H2/STO-3G/1.1/`)
3. Create a `Dataset`, and give it that pathname. It doesn't even take the list of attributes! It just scans the folder and gets any attributes there. This means that if say, yesterday you did `qml.data.load("qchem", molname="H2", basis="STO-3G", bondlength=1.1)` (which downloads the `_full.dat` file), but today you called that with `attributes=["molecule"]`, then did `my_dataset.list_attributes()`, you'd find that they'd all be there!

Ultimately, I left that implementation because we have lazy-loading implemented. While `my_dataset.hamiltonian` may be massive, it isn't actually saved to memory at all. This can be changed in the future if we don't like it, but that's the state of things today.

tl;dr specifying `attributes` changes what is _downloaded_, and while not direct, that does have some effect on what is then loaded to memory.